src/components/Profile/ProfileComponent.jsx

```diff
@@ -116,13 +124,46 @@ const ProfileComponent = ({ contactProfile }) => {
             setInputValue={setNickname}
             edit={edit}
           />
+          <ProfileInputField
+            inputName="WebId"
+            inputValue={webId}
+            endAdornment={
+              <IconButton
+                aria-label="Copy WebId"
+                edge="end"
+                onClick={() => {
+                  navigator.clipboard.writeText(webId);
+                  addNotification('success', 'webId copied to clipboard');
+                }}
```

Comment:
Think it's nice we got the clipboard now for these links.

Was wondering if we should convert this as a reusable function that could potentially be used on other pages like Contacts.

Reply:
Not a bad idea. Would be a good first issue for someone.